pyanaconda/modules/payloads/source/harddrive/harddrive.py

```diff
@@ -180,10 +180,10 @@ def tear_down_with_tasks(self):
         :return: list of tasks required for the source clean-up
         :rtype: [Task]
         """
-        tasks = []
-        if self.is_iso_mounted:
```

Comment:
Please remove whole `is_iso_mounted`. This was it's only use.

Reply:
Fixed.